frontend/src/services/usersService.js

```diff
@@ -1,7 +1,7 @@
 import { get, getApiUrl, post, patch } from './config';
 
 const getRecommendedUsers = () => get({ url: getApiUrl(`api/users/recommended/`) });
-const getUsersByKeyword = (keyword) => get({ url: getApiUrl(`api/users${keyword}/`) });
+const getUsersByKeyword = (keyword) => get({ url: getApiUrl(`api/users/?q=${keyword}`) });
```

Comment:
does this still work when keyword is an empty string?  If you clear the search input and click the users button on search, does it still display records?